src/Native/Runtime/assert.h

```diff
@@ -54,10 +54,21 @@ void Assert(const char * expr, const char * file, unsigned int line_num, const c
 
 #define FAIL_FAST_GENERATE_EXCEPTION_ADDRESS 0x1
 
-#define RhFailFast()  RhFailFast2(NULL, NULL)
+void RhFailFast();
 
 #define RhFailFast2(pExRec, pExCtx) \
 { \
     ASSERT_UNCONDITIONALLY("FailFast"); \
     PalRaiseFailFastException((pExRec), (pExCtx), (pExRec)==NULL ? FAIL_FAST_GENERATE_EXCEPTION_ADDRESS : 0); \
 }
+
+#define FATAL_ASSERT(e, msg) \
```

Comment:
`FATAL_ASSERT` is not very a good name - for a globally visible construct. It may be better to introduce another flavor of RhFailFast.